src/utils/utils.js

```diff
@@ -0,0 +1,3 @@
+export default function getRequestParams(req) {
+  return ['POST', 'PUT'].includes(req.method) ? req.body : req.query;
```

Comment:
There can be cases when post methods can also have query params, also there are couple of other HTTP methods with body: PATCH, DELETE and other

I suggest to log query and body for all possible methods. If body does not exist, then log null in this case
```
{
   body: req.body || null,
   query: req.query || null
}

Reply:
Thanks. Fixed.